data/campaigns/tutorial/scenarios/02_Tutorial_part_2.cfg

```diff
@@ -871,10 +871,9 @@ A full list of abilities and weapons specials, along with traits, may be found i
         [filter]
             side=1
             race=elf
-            level=2
-            [or]
-                level=3
-            [/or]
```

Comment:
We could, or we could jump in our time machines to 2017 where that syntaxt actually
already works :)